core/templates/dev/head/app.js

```diff
@@ -1220,3 +1220,20 @@ oppia.factory('codeNormalizationService', [function() {
     }
   };
 }]);
+
+// Service for retrieving algorithm id of classifier for a given interaction id.
+oppia.factory('algorithmIdRetrievalService', [function() {
+  // Maintain the Interaction-Classifier mapping as defined in feconf.
+  var interactionClassifierMapping = {
```

Comment:
@prasanna08 We could. So, do you suggest attaching a dict to the State object with the following structure: 
state.classifier_details = {
    'algorithm_id': <>,
    'classifier_data': <>
}